src/app/public/testing/phonefield/phone-field-fixture.ts

```diff
@@ -0,0 +1,143 @@
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+import {
+  SkyCountryFieldFixture
+} from '@skyux/lookup/testing';
+
+/**
+ * Provides information for and interaction with a SKY UX phone field component.
+ * By using the fixture API, a test insulates itself against updates to the internals
+ * of a component, such as changing its DOM structure.
+ */
+export class SkyPhoneFieldFixture {
+  private _debugEl: DebugElement;
+
+  // this property is lazy loaded and should be accessed via the private countryFixture property
+  private _countryFixture: SkyCountryFieldFixture;
```

Comment:
Code style > check your alphabetical ordering of properties. Also, public properties should come before private properties.